ktor-network/nix/src/io/ktor/network/sockets/DatagramSocketNative.kt

```diff
@@ -48,6 +48,7 @@ internal class DatagramSocketNative(
             }
         } catch (_: ClosedSendChannelException) {
         } catch (cause: IOException) {
+        } catch (cause: PosixException) {
```

Comment:
`PosixException` is throw further down in this file, so the exception must be caught here to prevent the app from crashing.